app/src/lib/get-os.ts

```diff
@@ -59,3 +59,7 @@ export const isMacOSBigSurOrLater = memoizeOne(
 export const isWindows10And1809Preview17666OrLater = memoizeOne(
   () => __WIN32__ && systemVersionGreaterThanOrEqualTo('10.0.17666')
 )
+
+export const isWindowsAndNoLongerSupportedByElectron = memoizeOne(
+  () => true || (__WIN32__ && systemVersionLessThan('10'))
```

Comment:
I understand this is here for testing, right? Still a bit dangerous 😂 I'll leave it as a "change requested" thing just in case 👀 
```suggestion
  () => __WIN32__ && systemVersionLessThan('10')
```

Reply:
This is why I hate working on auto update code 🤦